src/style.css

```diff
@@ -0,0 +1,99 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+nav {
+  display: flex;
+  gap: 8px;
+  background-color: #4cc9f0;
+  width: 100%;
+  height: 10vh;
+}
+
+nav h1,
+h2 {
+  margin-top: 10px;
+  margin-left: 10px;
+}
+
+.todo {
+  background-color: #fff;
+  width: 50%;
+  margin: auto;
+  border-radius: 0.25rem;
+  box-shadow: 5px 5px 5px #333;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 1rem;
+  color: #4cc9f0;
+}
```

Comment:
- [ ] There are a few areas that need to be arranged in order to match your design with the original one.  Kindly consider the layout structure as per the screenshot highlighted below

**Your Design**
![image](https://user-images.githubusercontent.com/50941074/206158441-b76bc66b-5ca7-45d1-a8f1-2b13ece9db74.png)

**The Original Design**
![image](https://user-images.githubusercontent.com/50941074/206158652-cce1c71f-4120-458d-b253-f56b6a98cae0.png)

I added these few styles below that if you add them, will give you what is required:
You can basically replace your styles with these styles above and it will give you a nice structure

```css
nav {
  display: flex;
  gap: 8px;
  background-color: #4cc9f0;
  width: 100%;
  height: 50px;
}

.todo {
  background-color: #fff;
  width: 50%;
  margin: auto;
  border-radius: 0.25rem;
  box-shadow: 1px 4px 30px rgba(0, 0, 0, 0.1);
  padding: 20px;
  margin-top: 60px;
}

.header {
  display: flex;
  justify-content: space-between;
  padding: 1rem 0;
  color: #4cc9f0;
}

.items {
  display: flex;
  justify-content: space-between;
}
```